sdk/python/kfp/v2/dsl/component_bridge.py

```diff
@@ -23,6 +23,7 @@
 from kfp.components._naming import _sanitize_python_function_name
 from kfp.components._naming import generate_unique_name_conversion_table
 from kfp.dsl import types
+from kfp.v2.dsl import container_op as v2_container_op
```

Comment:
Why do we need this alias? It seems we don't mix v1 and v2 together in file, so can this just be container_op?

Reply:
Initially I thought this can make readability better, but you're right, this is redundant. Removed.